mungegithub/submit-queue/deployment/nginx-redirect/nginx.conf

```diff
@@ -0,0 +1,41 @@
+events {
+  worker_connections  1024;
+}
+
+http {
+  server {
+    listen 80;
+    server_name contrib.submit-queue.k8s.io;
+    location / {
+        if ($http_user_agent ~ (GoogleHC) ) {
```

Comment:
That's the user-agent that the Google HTTP LoadBalancer sends for it's health checks.